tests/phpunit/tests/rest-api/rest-global-styles-controller.php

```diff
@@ -59,25 +77,97 @@ public static function wpSetupBeforeClass( $factory ) {
 				'tax_input'    => array(
 					'wp_theme' => 'tt1-blocks',
 				),
-			),
-			true
+			)
 		);
+
+		self::$post_id = $factory->post->create();
 	}
 
+	/**
+	 *
+	 */
+	public static function wpTearDownAfterClass() {
+		self::delete_user( self::$admin_id );
+		self::delete_user( self::$subscriber_id );
+	}
+
+	/**
+	 * @covers WP_REST_Global_Styles_Controller::register_routes
+	 */
 	public function test_register_routes() {
 		$routes = rest_get_server()->get_routes();
 		$this->assertArrayHasKey( '/wp/v2/global-styles/(?P<id>[\/\w-]+)', $routes );
+		$this->assertCount( 2, $routes['/wp/v2/global-styles/(?P<id>[\/\w-]+)'] );
+		$this->assertArrayHasKey( '/wp/v2/global-styles/themes/(?P<stylesheet>[^.\/]+(?:\/[^.\/]+)?)', $routes );
+		$this->assertCount( 1, $routes['/wp/v2/global-styles/themes/(?P<stylesheet>[^.\/]+(?:\/[^.\/]+)?)'] );
 	}
 
 	public function test_context_param() {
 		// TODO: Implement test_context_param() method.
-		$this->markTestIncomplete();
```

Comment:
```suggestion
		$this->markTestSkipped( 'Controller does not implement context_param().' );
```

Reply:
@spacedmonkey empty test methods get marked as "risky tests". Instead, mark them for skip with a message explaining why the test is skipped.